### PR TITLE
Updates Missing Links, Fixes MS Teams Bold

### LINKED ISSUE #93: MS Teams Bold Text - Needs an Updated Format
### Environment
* Python version:  3.9
* Nautobot version:  1.1.2
* nautobot-plugin-chatops version: 1.5.1

### Steps to Reproduce
1. Create a bolded text
2. Send to MS Teams

### Expected Behavior

Bolded Text

### Observed Behavior

Additional output including `*`

### LINKED ISSUE #95: Broken hyperlink
### Environment
* Python version:  3.8.10
* Nautobot version:  1.1.2
* nautobot-plugin-chatops version:  1.5.0

### Steps to Reproduce
1. Read Step 5 on this page - https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/docs/chat_setup/webex_setup.md#webex-setup

### Expected Behavior
Expected the hyperlink to be valid.

### Observed Behavior
The markdown hyperlink doesn't work.

### PR DESCRIPTION
**Pull request by jvanderaa**
- Updates missing link from docs
- Updates local link
- Fixes spelling that was found by VS Code Codespell
- Fixes MS Teams bolded format

Fixes #95 
Fixes #93

### CODE REVIEW

File: nautobot_chatops/dispatchers/ms_teams.py [hunk 1]
```diff
@@ -229,3 +229,7 @@ def user_mention(self):
         #   channel-and-group-conversations?tabs=python#adding-mentions-to-your-messages
         # For now we just use the user name as plaintext, not an actual mention
         return f"{self.context['user_name']}"
+
+    def bold(self, text):
+        """Mark text as bold."""
+        return f"*{text}*"
```
Comment (glennmatthews) [inline nautobot_chatops/dispatchers/ms_teams.py:235]: Does this fix #93?

File: docs/chat_setup/mattermost_setup.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
    - Copy the displayed url matching the Server you want to add the chatbot to.
      - Make sure to include the `http://` or `https://`.
      - If shown, make sure to include the port. Ex. `https://example.com:8065`
-6. Proceed to the [Nautobot Server Preparation and Configuration](./chat_setup.md#Nautobot Server Preparation and Configuration) section.
+6. Proceed to the [Nautobot Server Preparation and Configuration](./chat_setup.md#Plug-In-Installation) section.
```
Comment (glennmatthews) [inline docs/chat_setup/mattermost_setup.md:28]: FYI looks like there's also a broken link at step 4 above - mind fixing it as well?

**Review by glennmatthews — COMMENTED** [review]
Would it make sense to change the link text in each case from "Nautobot Server Preparation and Configuration" to "Plug-In Installation" to match the actual section header?

File: nautobot_chatops/dispatchers/ms_teams.py (hunk 1 above)
Comment (jvanderaa) [reply nautobot_chatops/dispatchers/ms_teams.py:235]: Ugg, looks like I have a collision in my branches!!!! I have a separate branch that I was preparing for that.

Comment (jvanderaa) [reply nautobot_chatops/dispatchers/ms_teams.py:235]: Yes it does. I will add that into here.

File: docs/chat_setup/mattermost_setup.md (hunk 2 above)
Comment (jvanderaa) [reply docs/chat_setup/mattermost_setup.md:28]: Thanks. I'll see if I can take a deeper dive through the docs again to check for it.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**